src/ui/simulator/toolbox/components/datagrid/renderer/mc-playlist.cpp

```diff
@@ -159,9 +162,14 @@ namespace Renderer
                 }
                 case MCPlaylistCol::WEIGHT:
                 {
-                    std::vector<int> yearsWeight = study->parameters.getYearsWeight();
+                    std::vector<float> yearsWeight = study->parameters.getYearsWeight();
                     assert(y < yearsWeight.size());
-                    return wxString::Format(wxT("%i"), yearsWeight[y]);
+
+                    std::ostringstream stream;
+                    stream << std::setprecision(3);
+                    stream << yearsWeight[y];
+
+                    return stream.str();
```

Comment:
why not just change the format to `"%.3f"` in the existing code ?

Reply:
This was used before : 
`return wxString::Format(wxT("%.3f"), yearsWeight[y])`
but this will display 100.00

With this stream it will display 100